front/src/app/LogsTable.tsx

```diff
@@ -1,9 +1,13 @@
+"use client";
```

Comment:
What is that line for? It is the first time I see this!

Reply:
It's NextJS syntax to state that the component and its children will be bundled to the client